### ISSUE TITLE
Setup GitHub Actions workflows for linting & formatting

### ISSUE THREAD

**Issue by christoff-linde** [SchizzY/next-month-web#7]
### Description

To ensure consistent formatting and try and keep consistent code quality, linting and formatting should be setup as `pre-commit`.

We could probably stick with the "standard" tools, namely:
- ESLint
- Prettier

~~However, BiomeJS does looks intriguing, so maybe we can give it a shot.~~ BiomeJS does not support `Svelte` yet, so sadge.

### Task List

- [x] Setup `pre-commit` hooks 
- [x] Setup linting & formatting configs

**Comment by SchizzY** [comment]
@christoff-linde if you'd like you can relook at this now.

**Comment by christoff-linde** [comment]
> @christoff-linde if you'd like you can relook at this now.

I'm wondering if this should just be local stuff actually. Using precommit

**Comment by SchizzY** [comment]
We could implement husky and call it a day running prettier and such?

**Comment by christoff-linde** [comment]
> We could implement husky and call it a day running prettier and such?

Yeah will maybe just do that. I have some configs backup up somewhere that we can use. Will make a PR
